vowpalwabbit/lda_core.cc

```diff
@@ -1234,7 +1234,7 @@ LEARNER::base_learner *lda_setup(vw &all)
 
   ld.decay_levels.push_back(0.f);
 
-  LEARNER::learner<lda> &l = init_learner(&ld, ld.compute_coherence_metrics ? learn_with_metrics : learn, 1 << all.weights.stride_shift());
+  LEARNER::learner<lda> &l = init_learner(&ld, ld.compute_coherence_metrics ? learn_with_metrics : learn, 1 << all.weights.stride_shift(), prediction_type::scalars);
```

Comment:
This seems to be working in this case.  My concern was in other reductions.  Since the `pred_type` doesn't need to be specified, this might actually be broken in some other spots.